src/drivers/amd/rembrandt/acp_hs_dma.c

```diff
@@ -302,9 +302,9 @@ static int acp_dai_hs_dma_get_data_size(struct dma_chan_data *channel,
 		tx_high = (uint32_t)io_reg_read(PU_REGISTER_BASE +
 				ACP_P1_HS_TX_LINEARPOSITIONCNTR_HIGH);
 		curr_tx_pos = (uint64_t)((tx_high<<32) | tx_low);
-		*free = (curr_tx_pos - prev_tx_pos) > hs_buff_size ? (curr_tx_pos - prev_tx_pos) % hs_buff_size : curr_tx_pos - prev_tx_pos;
 		prev_tx_pos = curr_tx_pos;
-		*avail = hs_buff_size - *free;
+		*free = (hs_buff_size / 2);
+		*avail = (hs_buff_size / 2);
```

Comment:
superfluous parentheses - in all of these

Reply:
parentheses are added for checkpatch compliance.